include/envoy/server/options.h

```diff
@@ -65,6 +65,11 @@ class Options {
   virtual const std::string& configPath() PURE;
 
   /**
+   * @return const std::string& the path to the v2 bootstrap file.
+   */
+  virtual const std::string& bootstrapPath() PURE;
```

Comment:
At least during the transition, we will be consuming both v1 JSON and v2 proto (in JSON format), and overlaying them, since the v2 proto handling isn't complete. So, I think for now this is probably the way to go.